forwards_paper.tex

```diff
@@ -560,7 +583,7 @@ \subsection*{Recording the pedigree in forwards time}
 process like Algorithm~\algref{W}.
 On the left is the tree showing all the edges output by the simulation,
 while on the right
-is the minimal tree representing the ancestry of the currently alive population.
+is the minimal tree representing the ancestry of the current generation of the population.
```

Comment:
hm, I actually like "currently alive" since it is exactly what we mean, while "generation" is ill-defined in an overlapping generation model